src/BackwardCompatiblityTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Axe.Windows.BackwardCompatiblityTest")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Axe.Windows.BackwardCompatiblityTest")]
```

Comment:
```suggestion
[assembly: AssemblyProduct("Axe.Windows.BackwardCompatibilityTest")]
```

Reply:
Renamed to Axe.Windows.OldFileVersionCompatibilityTests